internal/record/recording_https_proxy.go

```diff
@@ -262,7 +268,8 @@ func pumpWebsocket(src, dst *websocket.Conn, c chan []byte, quit chan int, prepe
 			quit <- 1
 			return
 		}
-		prefix := fmt.Sprintf("%s%d", prepend, cap(buf))
+		buf = append(buf, '\n')
+		prefix := fmt.Sprintf("%s%d", prepend, len(buf))
```

Comment:
Should we discount the newline from the payload size?

Reply:
I added the discount when replay the logs (extract chunks)